exporters/zipkin/src/main/java/io/opentelemetry/exporters/zipkin/ZipkinSpanExporter.java

```diff
@@ -380,6 +383,9 @@ protected Builder fromConfigMap(
      * @since 0.4.0
      */
     public ZipkinSpanExporter build() {
+      if (sender == null) {
+        sender = OkHttpSender.create(endpoint);
```

Comment:
I would prefer this to be inside the `setEndpoint()` method so we fail earlier in case of error

Reply:
Haha, we commented that. It will be in the same statement anyway. I prefer this way to ensure that setEndpoint always is the second option (I know that you can still achieve it with your suggestion but makes code a bit unclear)